webapp/src/pages/registration/components/RegistrationComponent.js

```diff
@@ -509,20 +509,14 @@ class RegistrationComponent extends Component {
 
                 {section.registration_questions
                   .sort((a, b) => a.order - b.order)
-                  .map(question => {
+                  .filter(question => {
```

Comment:
Hey man, what was the reason the filter was changed to filter + map?

Reply:
It wasn't 'filter' that was changed but 'map' was change to 'filter' + 'map'

> map calls a provided callback function but based on the logic implemented on the code the function only returns  value (map) if a certain condition is satisfied (filter) else nothing (Eslint warnings => map is expected to return a value)